hw6/hw6_task2.py

```diff
@@ -0,0 +1,15 @@
+
+def square_counting(side):
+    """
+    2. Write a function square that takes 1 argument, the side of the square, and returns 3 values (using a tuple):
+    the perimeter of the square, the area of the square, and the diagonal of the square.
+    """
+    diagonal = pow(side, 0.5)
+    perimeter = 4 * side
+    area = pow(side, 2)
+    square_tuple = (perimeter, area, diagonal)
+    return square_tuple
```

Comment:
you can just 
return perimeter, area, diagonal


python will automatically cast it to tuple